tools/reorder_docids.hpp

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include "app.hpp"
+#include "pisa/reorder_docids.hpp"
+
+namespace pisa {
+
+auto reorder_docids(ReorderDocuments args) -> int
+{
+    try {
+        if (args.bp()) {
+            return recursive_graph_bisection(RecursiveGraphBisectionOptions{
+                .input_basename = args.input_basename(),
```

Comment:
I don't think some of our compilers like ths format, could instead use `input_basename : args.input_basename(),` I think?

Reply:
Nah, I messed up the order + missed one member, just pushed a fix, it should work fine (let's hope).

Reply:
Yep, fixed it.